@here/harp-omv-datasource/lib/OmvTileInfoEmitter.ts

```diff
@@ -53,8 +53,8 @@ export class OmvTileInfoEmitter implements IOmvEmitter {
         techniques: IndexedTechnique[],
         featureId: number | undefined
     ): void {
+        const isGeoJSON = context.env.lookup("adapter") === "vt-json";
```

Comment:
maybe we should use something different than `isGeoJSON` here. Deciding how numbers get converted to integers is more general than vt-geojson. Also, we should probably avoid calling `context.env.lookup("adapter") === "vt-json";` for each feature, a property for required rounding behavior in `OmvDecoder.DecodeInfo`, initialized at construction time, may be less expensive to evaluate.

Reply:
For now, decided to pass the ``DataAdapter`` id through the ``DecodeInfo`` struct to avoid lookups, but the logic is still based on which adapter you use. We can maybe move to a more generic approach in the future, but for now we only need to do this to contrarrest geojson-vt lat2tile logic.